doc/client_lua_api.md

```diff
@@ -754,6 +760,16 @@ Call these functions only at load time!
     * returns reference to mod private `StorageRef`
     * must be called during mod load time
 
+### Mod channels
+You can find mod channels communication scheme in `docs/mod channels.png`.
+
+* `minetest.mod_channel_join(channel_name)`
+    * Client joins channel `channel_name`. If channel has no member it's created. You
+      should listen from incoming messages with `minetest.register_on_modchannel_message`
```

Comment:
can't register just to this channel?

Reply:
registration is a global table there is no object binding possibility currently, this can be enhanced later. Please help me to make this merged, and we can improve it together after that :)

Reply:
Yeah, this can be done latter actually